testing/acceptance_test.py

```diff
@@ -624,10 +624,9 @@ def join_pythonpath(*dirs):
         for p in search_path:
             monkeypatch.syspath_prepend(p)
 
-        os.chdir('world')
         # mixed module and filenames:
+        os.chdir('world')
```

Comment:
this code<> commend swap looks unintended

Reply:
actually I thought it seemed correct - there are two pytest runs here and for each we set up a suitable cwd folder

easy enough to revert if you dislike it, I'm just curious as to what the intended meaning here was then

Reply:
true, in full context it looks correct after all - before this change the lines where directly matching the code in question, but now its indeed marking the test sections instead